chia/util/streamable.py

```diff
@@ -59,7 +59,13 @@ def dataclass_from_dict(klass, d):
             return None
         return dataclass_from_dict(get_args(klass)[0], d)
     elif is_type_Tuple(klass):
-        return tuple(dataclass_from_dict(get_args(klass)[0], item) for item in d)
+        # Type is tuple, can have multiple different types inside
```

Comment:
I know @mariano54 was worried about this. I'm going to merge into main (but not into v.1.0.x branch) so we have more time to validate it but code in here is high risk.